src/core/dbc-gateway/generated/graphql.tsx

```diff
@@ -76,6 +76,8 @@ export enum AccessUrlType {
 
 export type Audience = {
   __typename?: "Audience";
+  /** PEGI age rating for games  */
+  PEGI?: Maybe<Pegi>;
```

Comment:
Is this just an updated version of the API, or do we now get PEGI for a specific reason?

Reply:
Yes, it is whatever is coming from the FBI graphql schema. I change a query and needed to regenerate the code.